dbfit-java/build.gradle

```diff
@@ -172,7 +191,7 @@ project('sqlserver') {
     description = 'DBFit sqlserver engine'
 
     dependencies {
-        compile group: 'net.sourceforge.jtds', name: 'jtds', version: '1.2.2'
+        //compile group: 'net.sourceforge.jtds', name: 'jtds', version: '1.2.2'
```

Comment:
I think we can merge the other ms sqlserver PR #366 first. And then rebase that on top of it - this line will like cause a small conflict. (Should be easy to handle I guess, just noting it).